examples/ballot.sol

```diff
@@ -0,0 +1,156 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity >=0.7.0 <0.9.0;
```

Comment:
This is not necessary for Solang.

Reply:
This is an unmodified version, directly from the solidity documentation. I thought it would be a neat showcase for us.

Reply:
The pragma can remain here. It has not effect for us. Maybe a comment to explain that is useful.